tests/Fixture/TestMigrationKernel.php

```diff
@@ -0,0 +1,125 @@
+
+        yield new ZenstruckFoundryBundle();
+
+        if (\getenv('USE_DAMA_DOCTRINE_TEST_BUNDLE')) {
+            yield new DAMADoctrineTestBundle();
+        }
+    }
+
+    protected function configureContainer(ContainerBuilder $c, LoaderInterface $loader): void
+    {
+        $c->loadFromExtension('framework', [
+            'http_method_override' => false,
+            'secret' => 'S3CRET',
+            'router' => ['utf8' => true],
+            'test' => true,
+        ]);
+
+        $c->loadFromExtension('zenstruck_foundry', [
+            'global_state' => [
+                GlobalStory::class,
+                GlobalInvokableService::class,
+            ],
+            'orm' => [
+                'reset' => [
+                    'mode' => AbstractORMPersistenceStrategy::RESET_MODE_MIGRATE,
+                ],
+            ],
+        ]);
+        $c->loadFromExtension('doctrine', [
+            'dbal' => ['url' => '%env(resolve:DATABASE_URL)%', 'use_savepoints' => true],
+            'orm' => [
+                'auto_generate_proxy_classes' => true,
+                'auto_mapping' => true,
+                'mappings' => [
+                    'Entity' => [
+                        'is_bundle' => false,
+                        'type' => 'attribute',
+                        'dir' => '%kernel.project_dir%/tests/Fixture/Entity',
+                        'prefix' => 'Zenstruck\Foundry\Tests\Fixture\Entity',
+                        'alias' => 'Entity',
+                    ],
+                    'Model' => [
+                        'is_bundle' => false,
+                        'type' => 'attribute',
+                        'dir' => '%kernel.project_dir%/tests/Fixture/Model',
+                        'prefix' => 'Zenstruck\Foundry\Tests\Fixture\Model',
+                        'alias' => 'Model',
+                    ],
+                ],
+                'controller_resolver' => ['auto_mapping' => true],
+            ],
+        ]);
+
+        // doctrine only handles different schema with postgres
+        if (str_starts_with(\getenv('DATABASE_URL') ?: '', 'postgresql')) {
+            $c->loadFromExtension('doctrine', [
+                'orm' => [
+                    'mappings' => [
+                        'Migrate' => [
+                            'is_bundle' => false,
+                            'type' => 'attribute',
+                            'dir' => '%kernel.project_dir%/tests/Fixture/EntityInAnotherSchema',
+                            'prefix' => 'Zenstruck\Foundry\Tests\Fixture\EntityInAnotherSchema',
+                            'alias' => 'Migrate',
+                        ],
+                    ],
+                ],
+            ]);
+        }
+
+        $c->loadFromExtension('doctrine_migrations', [
+            'migrations_paths' => [
+                'Zenstruck\\Foundry\\Tests\\Fixture\\Migrations' => '%kernel.project_dir%/tests/Fixture/Migrations',
+            ],
+            'transactional' => false,
```

Comment:
I also discovered that transactional mode + MySQL is buggy - at least in our tests, I'm not using MySQL currently, I can live with this, till a user comes with a problem around it